powerfulseal/policy/action_clone.py

```diff
@@ -25,6 +25,26 @@
 DEFAULT_TOXIPROXY_IMAGE = "docker.io/shopify/toxiproxy:2.1.4"
 DEFAULT_IPTABLES_IMAGE = "gaiadocker/iproute2:latest"
 
+class ModifyServiceAction:
+  """
+  Executes a http://jsonpatch.com/ object modify on a service
+  """
+  def __init__(self, type, name, namespace, k8s_inventory, logger=None):
+    self.type = type
+    self.name = name
+    self.namespace = namespace
+    self.k8s_inventory = k8s_inventory
+    self.logger = logger or makeLogger(__name__)
+
+  def execute(self):
+    body = [{"op": type, "path": "/spec/selector/chaos", "value": "true"}]
```

Comment:
```suggestion
    body = [{"op": self.type, "path": "/spec/selector/chaos", "value": "true"}]
```

It looks like you're using an undefined variable here.